.travis.yml

```diff
@@ -0,0 +1,15 @@
+os:
+  - linux
+dist: bionic
+language: python
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+
+if: type != pull_request  # do not build anything for PRs
```

Comment:
Actually, I'm having the following issue in cwl-WES: PRs from forks do not automatically trigger the pipeline, and I think that's because of this setting. At the same time, triggering the pipeline for PRs as well will trigger _two_ pipeline runs for internal PRs (because they are pushes _and_ PRs). Perhaps you could try to look for a solution, I think I've seen a discussion about this before, and I think there's probably (hopefully) a simple solution. I guess something along the lines of "run if no pull request OR (if pull request AND pull request comes from fork)"

Reply:
I'll see this :)

Reply:
@uniqueg I have a doubt. This is also a PR, so according to this condition  `if: type != pull_request  # do not build anything for PRs`  it shouldn't have a build, right?

Reply:
It's building for the push and not for the PR. That's the thing, if it's _also_ building for the PR, the pipeline runs twice

Reply:
https://docs.travis-ci.com/user/conditions-v1

There is a `fork` attribute known to Travis. If a PR is coming from a fork, it is set to `True`, so you can build a condition with that.

I think you could do something like this:
`if: (type == push) OR (type == pull_request AND fork == true)`

This way we are building for all pushes, as well as for PRs if they come from forks. Note that this is untested though. See here how to test Travis conditions: https://docs.travis-ci.com/user/conditions-testing

Reply:
I tested it with the following:

`travis-conditions eval "(type == push) OR (type == pull_request AND fork == true)" --data '{"type": "pull_request", "fork": "false"}'` -> `false`

`travis-conditions eval "(type == push) OR (type == pull_request AND fork == true)" --data '{"type": "push"}'` -> `true`

`travis-conditions eval "(type == push) OR (type == pull_request AND fork == true)" --data '{"fork": "true","type": "pull_request"}'` -> `true`

It should work.